contrib/completion/zsh/_docker

```diff
@@ -261,8 +262,12 @@ __docker_complete_log_options() {
     if compset -P '*='; then
         case "${${words[-1]%=*}#*=}" in
             (syslog-format)
-                syslog_format_opts=('rfc3164' 'rfc5424' 'rfc5424micro')
-                _describe -t syslog-format-opts "Syslog format Options" syslog_format_opts && ret=0
+                local opts=('rfc3164' 'rfc5424' 'rfc5424micro')
+                _describe -t syslog-format-opts "syslog format options" opts && ret=0
+                ;;
+            (mode)
+                local opts=('blocking' 'nonblocking')
```

Comment:
Hum sorry, I based my PR on @albers bash's completion (https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L793) but looks like it is indeed `non-blocking` (https://github.com/docker/docker/blob/master/api/types/container/host_config.go#L234)